server/index.js

```diff
@@ -145,13 +182,13 @@ app.post('/api/uploads', express.urlencoded(), uploadsMiddleware, (req, res, nex
       union
       select "ot"."tagId"
       from "tags" as "ot"
-      where "ot"."label" = any ($13::text[])
+      where "ot"."label" = any ($14::text[])
     ) as "t" on true
   )
   select * from "newEntry"
   `;
   const {
-    description, title, printer, totalFilamentUsed, timeToPrint,
+    userId, description, title, printer, totalFilamentUsed, timeToPrint,
```

Comment:
Most places where you are using `userId` will need to be updated to access it from `req.user` instead of `req.body` (assuming you have an `authorizationMiddleware`

Reply:
On the client side, you will need to include the `x-access-token` header. For instance, in the form for posting a new entry